easybuild/easyconfigs/v/Virtuoso-opensource/Virtuoso-opensource-7.2.6.1-foss-2021a.eb

```diff
@@ -0,0 +1,71 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'Virtuoso-opensource'
```

Comment:
Wouldn't it make more sense to use just `Virtuoso` as name?

Reply:
`Virtuoso-opensource` is the real name. `Virtuoso` is the name of the non-opensource version with a different set of features. So I keep the `Virtuoso-opensource` name